pkg/controller/kubelet-config/kubelet_config_controller.go

```diff
@@ -163,6 +186,10 @@ func kubeletConfigTriggerObjectChange(old, new *mcfgv1.KubeletConfig) bool {
 	return false
 }
 
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
can you remove this? i think this might have been when the features code was in here as well.

Reply:
agree.

Reply:
fixed